src/Loadouts.py

```diff
@@ -43,7 +43,7 @@ def get_match_loadouts(self, match_id, players, weaponChoose, valoApiSkins, name
                         if skin_id.lower() == skin["uuid"].lower():
                             rgb_color = self.colors.get_rgb_color_from_skin(skin["uuid"].lower(), valoApiSkins)
                             # if rgb_color is not None:
-                            weaponLists.update({players[player]["Subject"]: color(skin["displayName"], fore=rgb_color)})
+                            weaponLists.update({players[player]["Subject"]: color(skin["displayName"].rsplit(" ")[0], fore=rgb_color)})
```

Comment:
This is from https://github.com/isaacKenyon/VALORANT-rank-yoinker/blob/38ba8e080655e1285510e780b99e1194c7a269eb/src/Loadouts.py#L46 I really like that it made it show the skin name only. Plus now the user will know what weapon it is going to show so no need to show what weapon it belongs to.